config/cors.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+return [
+
+    /*
+     |--------------------------------------------------------------------------
+     | Laravel CORS Defaults
+     |--------------------------------------------------------------------------
+     |
+     | The defaults are the default values applied to all the paths that match,
+     | unless overridden in a specific URL configuration.
+     | If you want them to apply to everything, you must define a path with *.
+     |
+     | allowedOrigins, allowedHeaders and allowedMethods can be set to ['*']
+     | to accept any value, the allowed methods however have to be explicitly listed.
+     |
+     */
+    'defaults' => [
+        'supportsCredentials' => true,
+        'allowedOrigins'      => [],
+        'allowedHeaders'      => [],
```

Comment:
I guess we need to include `X-Cachet-Token` here to authenticate the API based on the package readme.